ingest/bin/post_process_metadata.py

```diff
@@ -0,0 +1,91 @@
+#! /usr/bin/env python3
+
+"""Reformat pandas DataTables for a pathogen build.
+
+Expecting one argument, a NCBI Virus metadata.tsv file
+"""
+# ===== Dependencies
+import argparse
+import os
+import sys
+
+import numpy as np
+import pandas as pd
+
+
+def parse_args():
+  # Main help command
+  parser = argparse.ArgumentParser(
+    description = "USAGE: python post_process_metadata.py --metadata 'data/metadata.tsv'"
+  )
+  # Add first argument
+  parser.add_argument(
+    "--metadata",
+    help = "NCBI Virus metadata.tsv file.",
+    required = True
+  )
+  parser.add_argument(
+    "--outfile", default="processed_metadata.tsv",
+    help = "Processed metdata file name. [default: processed_metadata.tsv] ",
+    required = False
+  )
+  
+  return parser.parse_args()
+
+# === Private methods
+
+def _set_strain_name(record):
+  """Check Isolate_s and Strain_s to find the strain name"""
+  strain_name=""
```

Comment:
Based on the logic below, I wouldn't expect this value to ever get used. Have you run into a specific case where it did?

Reply:
Excess caution after dealing with WDL weirdness in an if/else. I'll take it out here, agree that python should be fine.